### ISSUE TITLE
Consul and Vault URLs aren't correct

### ISSUE THREAD

**Issue by nadenf** [hashicorp/terraform-provider-hcp#124]
### Terraform Version and Provider Version
```
0.15.3
```

### Affected Resource(s)

* hcp_consul_cluster
* hcp_vault_cluster

### Terraform Configuration Files

```hcl
output consul_url {
  description = ""
  value = hcp_consul_cluster.consul.consul_private_endpoint_url
}

output vault_url {
  description = ""
  value = hcp_vault_cluster.vault.vault_private_endpoint_url
}
```

### Expected Behavior

URL outputs should be actual URLs i.e. prefixed with https://

### Actual Behavior

URLs such as the following are generated:

```
consul.private.consul.xxxxxxxxxx.aws.hashicorp.cloud
```

Which then cause errors when you pass them into the Consul and Vault providers.

**Comment by bcmdarroch** [comment]
Hi @nadenf! Thanks for adding this issue. I'll bring this request up to our cloud Consul and Vault teams and provide an update when I hear from them.

**Comment by nadenf** [comment]
Closed by #145
